src/plugins/kibana/public/dashboard/components/panel/panel.html

```diff
@@ -1,6 +1,10 @@
 <div class="panel panel-default" ng-switch on="panel.type" ng-if="savedObj || error">
   <div class="panel-heading">
-    <span class="panel-title">{{savedObj.title}}</span>
+    <span class="panel-title">
+      <i class="fa" ng-class="savedObj.vis.type.icon"
+        aria-label="{{savedObj.vis.type.title}} Icon" title="{{savedObj.vis.type.title}}"></i>
```

Comment:
nitpicky...
Since these attributes span multiple lines, shouldn't each attribute be on it's own line?